scripts/store-triggers.R

```diff
@@ -0,0 +1,32 @@
+# Write on-trigger yaml
+# This script stores what the 'on' triggers look like for each github action
```

Comment:
This is the bad script that stores whatever is in the current version of the gha which requires you to turn all the github actions ON before running this.